app/services/gov_one_auth_service.rb

```diff
@@ -0,0 +1,88 @@
+class GovOneAuthService
+  # @param code [String]
+  def initialize(code)
+    @code = code
+  end
+
+  # @return [Hash]
+  def tokens
+    body = {
```

Comment:
private method?

Reply:
I call this method in the `omniauth_callbacks_controller`

Reply:
I was thinking how we could reduce method size, so extracting `body` into a private method for example. Would that be possible?